lightning/src/chain/onchaintx.rs

```diff
@@ -481,6 +481,54 @@ impl<ChannelSigner: WriteableEcdsaChannelSigner> OnchainTxHandler<ChannelSigner>
 		events.into_iter().map(|(_, event)| event).collect()
 	}
 
+	/// Triggers rebroadcasts/fee-bumps of pending claims from a force-closed channel. This is
+	/// crucial in preventing certain classes of pinning attacks and ensures reliability if
+	/// broadcasting fails. We recommend invoking this roughly once per minute, though it doesn't
+	/// have to be perfect.
+	pub(crate) fn rebroadcast_pending_claims<B: Deref, F: Deref, L: Deref>(
+		&mut self, current_height: u32, broadcaster: &B, fee_estimator: &LowerBoundedFeeEstimator<F>,
+		logger: &L,
+	)
+	where
+		B::Target: BroadcasterInterface,
+		F::Target: FeeEstimator,
+		L::Target: Logger,
+	{
+		let mut bump_requests = Vec::with_capacity(self.pending_claim_requests.len());
+		for (package_id, request) in self.pending_claim_requests.iter() {
+			let inputs = request.outpoints();
+			log_info!(logger, "Triggering rebroadcast/fee-bump for request with inputs {:?}", inputs);
+			bump_requests.push((*package_id, request.clone()));
+	   }
+		for (package_id, request) in bump_requests {
+			self.generate_claim(current_height, &request, false /* manually_bump_feerate */, fee_estimator, logger)
+				.map(|(_, new_feerate, claim)| {
+					if let Some(mut_request) = self.pending_claim_requests.get_mut(&package_id) {
+						mut_request.set_feerate(new_feerate);
+					}
+					match claim {
+						OnchainClaim::Tx(tx) => {
+							log_info!(logger, "Broadcasting RBF-bumped onchain {}", log_tx!(tx));
```

Comment:
is “RBF-bumped” not dependent now on `manually_bump_feerate`, like a transaction can be rebroadcast/re-sign still with the same feerate.